idea_town/frontend/static-src/app/views/base-view.js

```diff
@@ -12,11 +12,6 @@ export default AmpersandView.extend({
     return mustache.render(this._template, ctx);
   },
 
-  remove() {
```

Comment:
As far as I could tell, [it's redundant](https://github.com/AmpersandJS/ampersand-view/blob/master/ampersand-view.js#L160) and missing some of the out-of-box housekeeping.